lib/components/artwork_grids/infinite_scroll_grid.js

```diff
@@ -176,7 +192,9 @@ class InfiniteScrollArtworksGrid extends React.Component {
     const spacerStyle = {
       height: this.props.itemMargin,
     }
-    const artworks = this.props.artist.artworks ? this.props.artist.artworks.edges : []
+
+    const queryKey = this.props.queryKey
+    const artworks = this.props[queryKey].artworks ? this.props[queryKey].artworks.edges : []
```

Comment:
I’d like to see the artwork accessing refactored into a method:

```js
artworks() {
  return this.props[this.props.queryKey].artworks
}
```

Not a blocker, though.